eng/pipelines/variables/common.yml

```diff
@@ -19,5 +19,11 @@ variables:
 - name: testScriptPath
   value: ""
 - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-    - name: build.imageBuilderDockerRunExtraOptions
-      value: -e DOCKER_REPO=$(acr.server)/$(stagingRepoPrefix)dotnet-buildtools/prereqs
+  - name: build.imageBuilderDockerRunExtraOptions
+    value: -e DOCKER_REPO=$(acr.server)/$(stagingRepoPrefix)dotnet-buildtools/prereqs
+- ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/production') }}:
+  - name: latestTag
+    value: --var LatestTag=latest
+- else:
+  - name: latestTag
+    value: --var LatestTag=staging
```

Comment:
The use of `latestTag` as the variable name is a bit confusing.

1. It's not the full tag, just the suffix
2. It's not always the `latest` tag pending the branch

How about naming it more literally like `FloatingTagSuffix`?

Reply:
That's a good point. I've changed it to your suggestion

Reply:
Done